x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/preserve_layout.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import path from 'path';
+import { Size } from '../../../../../types';
+import { Layout, PageSizeParams } from './layout';
+
+const ZOOM: number = 2;
```

Comment:
Zoom used to be a param that could be passed in from a caller, but 2 was the default. This makes it completely hardcoded.

Also, you've removed the explanatory comments: https://github.com/elastic/kibana/blob/master/x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/preserve_layout.js#L9-L12

Reply:
Intentional. This setting was never exposed to the user as configurable - https://github.com/elastic/kibana/issues/20091

I don't think we ever will expose it. The only reason we use it is to make sure the resolution is a little bigger on the screenshots.

From Brandon in that above issue: 

> The zoom is an undocumented setting that I intended to remove, but never got around to it.